meta-balena-common/recipes-connectivity/networkmanager/balena-files/90shared

```diff
@@ -58,11 +58,21 @@ then
   exit 0
 fi
 
-# Safeguard, this should never happen
-# Exactly 0 or 1 rule should match, bail out if there are more & investigate
-if [ "$(echo "${FW_RULE_ARGS}" | wc -l)" -gt 1 ]
+# Sometimes on NetworkManager restart a new rule is added
+# but the old one is not properly cleand up
+# Remove the duplicates here as the rules are all the same
+DUPS=0
+while [ "$(echo "${FW_RULE_ARGS}" | wc -l)" -gt 1 ]
+do
+  DUPS=$(("${DUPS}" + 1))
```

Comment:
nitpick: This could be ` DUPS=$(( DUPS + 1 ))` which reads cleaner

Reply:
Hm, TIL you can do that. I had `$["${DUPS}" + 1]` and changed it to `$(( ))` because shellcheck was complaining. Since this already has the tests running, I'll let them finish as it is. I'll redo it if the PR does not merge and needs a rebase or retry.